action.php

```diff
@@ -202,8 +202,11 @@ public function handle_loginform(Doku_Event &$event, $param) {
         $form =& $event->data;
         $html = '';
 
-        if ($this->getConf('mailRestriction') !== '') {
-            $html .= sprintf($this->getLang('eMailRestricted'),$this->getConf('mailRestriction'));
+        $validDomains = $hlp->getValidDomains();
+
+        if ($validDomains[0] !== '') {
```

Comment:
You're right. It exists but is undocumented. See config.class.php line 623. Can you document it?